docs/pages/cli/config.mdx

```diff
@@ -1 +1,94 @@
-# Config
+# The World config
+
+Certain CLI commands, such as [`mud tablegen`](/cli/tablegen) and [`mud worldgen`](/cli/worldgen) require the MUD configuration file.
+This file needs to be named `mud.config.ts` and be in the same folder as your `foundry.toml` file.
+
+The config is used to define:
+
+- The tables in your project in the `tables` object of your configuration.
+  See [the `Store` documentation](/store/tables) for more information on table and their format.
+- The [namespace](/world/namespaces-access-control) the [`System`s](/world/systems) and tables will be deployed in.
+- The `System`s in your project.
+  By default, the deployer will find all Solidity matching `*System*.sol` (so any file containing `System` with a `.sol` extension, in any folder) and deploy them as `System`.
+  If you want greater control over your systems (to change their public access or their name), you can use the `systems` object in the config
+- The [modules](/world/modules) that will be installed on the `World`.
+
+An example of a `World` config with all the different configuration keys and options used:
+
+```tsx
+import { mudConfig } from "@latticexyz/world/register";
+import { resolveTableId } from "@latticexyz/config";
+
+export default mudConfig({
+  excludeSystems: ["System3", "System2"],
+  worldContractName: "CustomWorld",
+  namespace: "mud",
+  systems: {
+    IncrementSystem: {
+      name: "increment",
+      openAccess: true,
+    },
+  },
+  tables: {
+    CounterTable: {
+      valueSchema: {
+        value: "uint32",
+      },
+    },
```

Comment:
should we add all table options here?

Reply:
On this page? Definitely - my bad. In the example? No. It's best to keep the code sample simple.